app/views/retro/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="nav">
+  <div class="headerItems">
+    <Button id="start-new-button" class="start-button">Post your topics!</Button>
+    <Button id="time"></Button>
+    <%= image_tag("/assets/wye-logo.png", size: "40x40", class: "header-icon") %>
+  </div>
+</div>
+<div class="boards">
+  <fieldset class="section" id="mention">
+    <legend>To mention</legend>
+    <textarea class="txtarea" id="txt0" placeholder="To mention.." rows='1'></textarea>
+  </fieldset>
+  <fieldset class="section" id="discuss">
+    <legend>To Discuss</legend>
+  </fieldset>
+</div>
```

Comment:
Why do you just write html and do not use [helpers provided by ActionView](https://edgeguides.rubyonrails.org/action_view_overview.html)?

They are great and make the code more readable, and in the end it will be html like this 😄

Reply:
No reason in particular XD I thought about refactoring the html files, because I use some helpers but some html tags in other places 🤷‍♀️ 

In particular, I think I just forgot about it.